src/axom/quest/examples/CMakeLists.txt

```diff
@@ -326,13 +326,20 @@ if((CONDUIT_FOUND OR
         )
 
     if(AXOM_ENABLE_TESTS AND AXOM_DATA_DIR)
-        # 3D shaping tests.  No 2D shaping in this feature, at least for now.
-        set(_nranks 1)
+
+        if(AXOM_ENABLE_MPI)     # this test requires MPI when available
+            set(_nranks 1)      # (but only 1 rank)
+        else()                  # but can otherwise be run without MPI
+            unset(_nranks)
+        endif()
 
         # Run the in-memory shaping example on with each enabled policy
         set(_policies "seq")
         if(RAJA_FOUND)
-            blt_list_append(TO _policies ELEMENTS "omp"  IF AXOM_ENABLE_OPENMP)
+            # Hanging issue on Windows
+            if(NOT WIN32)
+                blt_list_append(TO _policies ELEMENTS "omp"  IF AXOM_ENABLE_OPENMP)
+            endif()
```

Comment:
Do you have more information about the hang?
Could/should this be fixed?
Should we create an issue for it?

Reply:
Thanks -- do you think we should update vcpkg to RAJA@develop after the RAJA PR merges? Or should we add a patch to our RAJA?

Since we maintain our own vcpkg RAJA package, I think either option should be ok, and it'll be good to fix our Windows TPL CI.

Reply:
> Thanks -- do you think we should update vcpkg to RAJA@develop after the RAJA PR merges? 

I think the first/former option is better. 
At first, I wasn't certain if it was possible for vcpkg to track a commit instead of a branch or tag (wasn't reading the documentation correctly :satisfied:)